spec/graphql/types/user_type_spec.rb

```diff
@@ -44,5 +48,27 @@
       )
       expect(organizations.count).to eq(16)
     end
+
+    context 'filters by role' do
+      it 'returns the organizations with the role: read' do
+        role = 'read'
+        organizations = organizations_field.resolve(
+          user,
+          organizations_field.default_arguments('role' => role),
+          {}
+        )
+        expect(organizations.count).to eq(20)
+      end
+
+      it 'returns the organizations with the role: admin' do
+        role = 'admin'
+        organizations = organizations_field.resolve(
+          user,
+          organizations_field.default_arguments('role' => role),
+          {}
+        )
+        expect(organizations.count).to eq(1)
+      end
+    end
```

Comment:
Why not
```ruby
    let(:organizations) do
      organizations_field.
        resolve(user,
                organizations_field.default_arguments('role' => role),
                {})
    end
    context 'filters by role' do
      context 'read' do
        let(:role) { 'read' }
        it 'returns the organizations with the corresponding role' do
          expect(organizations.count).to eq(20)
        end
      end

      context 'admin' do
        let(:role) { 'admin' }
        it 'returns the organizations with the corresponding role' do
          expect(organizations.count).to eq(1)
        end
      end
    end
```
I find this much easier to read and maintain because you DRY the complicated stuff.